README.md

```diff
@@ -0,0 +1,14 @@
+# usbd-ethernet
+
+Ethernet-over-USB implementation using CDC-EEM USB class. Designed for
+[usb-device](https://crates.io/crates/usb-device).
+
+CDC-EEM is an open standard (defined by USB-IF) designed for Ethernet emulation,
+i.e., when there is no physical Ethernet link, and solely USB is used for
+transmitting frames. CDC-EEM is supported out of the box on Linux and FreeBSD.
+
+TBD: does macOS and Windows support it?
```

Comment:
@arturkow2000 Can we link to the documentation on fobnail.3mdeb.com which explains how we researched the driver topic?

Reply:
@macpijan You mean to link to this https://github.com/fobnail/docs/blob/211bccec51d0329e02b785cef51439d39ff62b9a/docs/eth-over-usb-research.md?

Reply:
@arturkow2000 Yes. Please link it in a new MR.